spec/tests/require.fmf

```diff
@@ -78,6 +88,16 @@ example:
         ref: devel
         nick: openssl
         name: /certgen
+        type: library
+
+  - |
+    # Require local files needed for the library or test
+    require:
+      - type: file
+        pattern:
+          - /include
+          - /Library/common/helper.sh
+          - /files/photos/IMG*
```

Comment:
This is more globbing example than regexp. What will be supported? If regexp then more fitting example would be `IMG.*`

Personally globbing feels more natural.

Reply:
I guess we ended up deciding on regexp. Personally would prefer glob, but not that important IMO. Will change the example.

Reply:
Yes, the hacking session discussion about glob versus regexp ended up with regexp to keep consistency (they are already used for filtering).